### PR TITLE
feat(qvst): Qvst part

### LINKED ISSUE #56: Front - Android - QVST Campaign feature

### PR DESCRIPTION
**Pull request by mmarchal**
# Description
What are changes related to ?

# Linked Issues
What are the issues fixed by this pull request ?

# Changes
What does it change ?
Is is a breaking change ?
Is is a new feature ?
Is is a patch, fix, hotfix ?

# Screenshots
Put screenshots (if any)

# Tests
How has it been tested ?

- [ ] Manually
- [ ] Automated tests
- [ ] QA
- [ ] Other

# Additional information
Precise any other information

### CODE REVIEW

File: app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstAnswer.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.xpeho.xpeapp.data.model.qvst
+
+/*
+{
+                "answer": "Jamais",
+                "value": "1"
+            }
+ */
```
Comment (MayuriXx) [inline app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstAnswer.kt]: here delete it

File: app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstQuestion.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            },
+            {
+                "id": "5",
+                "answer": "Pas du tout",
+                "value": "1"
+            }
+        ]
+    }
+ */
+
```
Comment (MayuriXx) [inline app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstQuestion.kt]: here delete it

File: app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstTheme.kt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.xpeho.xpeapp.data.model.qvst
+
+/*
+{
+            "id": "8",
+            "name": "La santé au travail"
+        }
+ */
```
Comment (MayuriXx) [inline app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstTheme.kt]: here delete it

File: app/src/main/java/com/xpeho/xpeapp/data/service/WordpressService.kt [hunk 4]
```diff
@@ -3,15 +3,17 @@ package com.xpeho.xpeapp.data.service
...
 import okhttp3.OkHttpClient
 import okhttp3.logging.HttpLoggingInterceptor
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
-import retrofit2.http.Body
-import retrofit2.http.Headers
-import retrofit2.http.POST
+import retrofit2.http.*
 
+//private const val BASE_URL = "http://10.0.2.2:8000/wp-json/"
```
Comment (MayuriXx) [inline app/src/main/java/com/xpeho/xpeapp/data/service/WordpressService.kt]: here delete it

File: app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+            textAlign = TextAlign.Center,
+            fontWeight = FontWeight.Bold,
+            fontStyle = FontStyle.Italic,
+        ),
+        modifier = Modifier
+            .padding(32.dp),
+    )
+    if (questionsAnsweredSize == questionsSize) {
+        Text(
+            text = "Vous avez répondu à toutes les questions",
```
Comment (MayuriXx) [inline app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt]: here

File: app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+                textAlign = TextAlign.Center,
+                fontWeight = FontWeight.Bold,
+                fontStyle = FontStyle.Italic,
+            ),
+            modifier = Modifier
+                .padding(32.dp),
+        )
+    } else {
+        Text(
+            text = "Vous avez répondu à $questionsAnsweredSize questions sur $questionsSize",
```
Comment (MayuriXx) [inline app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt]: here

File: app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+                        questionInDialog.value = question
+                        showDialog.value = true
+                    }
+                )
+            }
+        }
+    }
+    if (questionsAnswered.isNotEmpty()) {
+        QvstTitleHeaderComposable(
+            title = "Questions répondues",
```
Comment (MayuriXx) [inline app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt]: here

File: app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt [hunk 8]
```diff
@@ -0,0 +1,289 @@
...
+        Column(
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            QuestionTextComposable(
+                question = question.question
+            )
+            if (isAnswered && question.userAnswer != null) {
+                Text(
+                    text = "Réponse: ${question.userAnswer}",
```
Comment (MayuriXx) [inline app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt]: here

File: app/src/main/java/com/xpeho/xpeapp/ui/componants/qvst/QvstCampaignList.kt [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+            id = "1",
+            name = "Campagne 1",
+            theme = QvstTheme(id = "1", name = "Thème 1"),
+            status = "OPEN",
+            start_date = "2021-09-01",
+            end_date = "2021-09-30",
+            participation_rate = "0"
+        ),
+    )
+}
```
Comment (MayuriXx) [inline app/src/main/java/com/xpeho/xpeapp/ui/componants/qvst/QvstCampaignList.kt]: here

File: app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstAnswer.kt (hunk 1 above)
Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstAnswer.kt]: Corrected !

File: app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstQuestion.kt (hunk 2 above)
Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstQuestion.kt]: Corrected !

File: app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstTheme.kt (hunk 3 above)
Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/data/model/qvst/QvstTheme.kt]: Corrected !

File: app/src/main/java/com/xpeho/xpeapp/data/service/WordpressService.kt (hunk 4 above)
Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/data/service/WordpressService.kt]: Corrected !

File: app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt (hunk 5 above)
Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt]: Corrected !

File: app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt (hunk 6 above)
Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt]: Corrected !

File: app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt (hunk 7 above)
Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt]: Corrected !

File: app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt (hunk 8 above)
Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/ui/page/qvst/QvstComponants.kt]: Corrected !

File: app/src/main/java/com/xpeho/xpeapp/ui/componants/qvst/QvstCampaignList.kt (hunk 9 above)
Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/ui/componants/qvst/QvstCampaignList.kt]: Corrected !

**Review by MayuriXx — APPROVED** [review]

**Pull request merged**